site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/repository/SiteStore.java

```diff
@@ -39,17 +39,20 @@ public SiteStore(FhirRepository repository,
      */
     @Override
     public String saveEntity(Site entity) {
-
-        // TODO: Check if the Organization already exists.
+        String orgId = "";
 
         Organization org = converter.convert(entity);
-        String orgId = repository.saveOrganization(org);
+        orgId = repository.saveOrganization(org);
         org.setId(orgId);
-        logger.info(FhirRepo.SAVE_REQUEST.message(), orgId);
+        if (logger.isInfoEnabled()) {
+            logger.info(FhirRepo.REQUEST_PAYLOAD.message(), orgId);
+        }
```

Comment:
Isn't this anyway logged on the "saveOrganization" method?

Reply:
Removed the logs

Reply:
This has been fixed